main.tf

```
@@ -16,7 +16,7 @@ resource "azurerm_key_vault" "secrets" {
   name                       = "${var.key_vault_name_prefix}-${random_string.key_vault_suffix.result}"
   tags                       = local.tags
   location                   = var.azure_location
```

Please consider using `location` as a variable name. This matches our default and best practice to use the property where the variable is used as a variable name, when possible.

```suggestion
  location                   = var.location
```